lib/validators/default-values-validator.ts

```diff
@@ -90,10 +90,16 @@ export class DefaultValuesValidator extends Validator {
         return true;
     }
 
+    private validateNumberValue(property: ComponentProperty): boolean {
```

Comment:
Should this be named `validateDefaultValue`?

Reply:
Hmm, I think it is already clear this is about default values? This validates the default value is a number. The other two functions validate if they are a string or object. It depends on the control type which values are supported.

Reply:
The way I read the function name is that it validates that a value is a number while in reality it validates that the default value is a number. But that same goes for the other functions that I didn't see, so let's leave it like this.